src/init/mod.rs

```diff
@@ -0,0 +1,17 @@
+use std::fs;
+use std::fs::{File, OpenOptions};
+use std::io;
+use std::io::prelude::*;
+use std::os::unix;
+use std::path::Path;
+
+pub fn new_profile(name: String) {
+    println!("create profile {}", name);
+}
+
+fn mkdir(name: String) {
+    match fs::create_dir(name) {
+        Err(why) => println!("! {:?}", why.kind()),
+        Ok(_) => (),
```

Comment:
This usage of the error returned by `create_dir` could be much more succinctly expressed with `fs::create_dir(name).unwrap();`